sdk/eventhub/azure-eventhub/CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 # Release History
 
-## 5.2.0b2 (Unreleased)
+## 5.2.0 (2020-09-07)
 
+**New Features**
+
+- Connection strings used with `from_connection_string` methods now supports using the `sharedaccesstoken` key in leiu of `sharedaccesskey` and `sharedaccesskeyname`, taking the string of the properly constructed token as value.
```

Comment:
Suggest `sharedaccesssignature` because the connection string uses it.

Reply:
Oop, it initially was token, but when we changed it as a group I clearly missed replacing it here.  Good catch.